geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionClearDUnitTest.java

```diff
@@ -430,13 +423,54 @@ public void normalClearFromClient() {
         .isEqualTo(0);
   }
 
+  @Test
+  public void testClearTime() {
+    configureServers(false, true);
+    client1.invoke(this::initClientCache);
+    client2.invoke(this::initClientCache);
+
+    accessor.invoke(() -> feed(false));
+    verifyServerRegionSize(NUM_ENTRIES);
+    dataStore1.invoke(() -> {
+      PartitionedRegion partitionedRegion = (PartitionedRegion) getRegion(false);
+      assertThat(partitionedRegion.getCachePerfStats().getClearTime()).isEqualTo(0L);
```

Comment:
clearTime maybe not always 0. It's better to set a expected range, such as less than 10ms.